unity_jobs_simulator/Assets/Scripts/Robot_Controller.cs

```diff
@@ -116,6 +126,11 @@ public void initializeJoints()
         {
             targetPos = new List<float>() {0, 0, 0, 0, 0, 90};
         }
+        if(Input.GetKeyDown(KeyCode.T))
```

Comment:
How is this used? I'm not sure purpose this is serving. Was this used for debugging?